mantidimaging/gui/windows/recon/view.py

```diff
@@ -381,3 +394,7 @@ def open_help_webpage(self, page: str):
 
     def change_refine_iterations(self):
         self.refineIterationsBtn.setEnabled(self.algorithm_name == "SIRT_CUDA")
+
+    def on_change_colour_palette(self):
+        change_colour_palette = PaletteChangerView(self, self.hists, self.image_view.projection.image)
```

Comment:
Can you move this all to work on the reconstructed view `self.image_view.recon.image`